haas/cli.py

```diff
@@ -295,7 +295,9 @@ def serve(port):
     # We need to import api here so that the functions within it get registered
     # (via `rest_call`), though we don't use it directly:
     from haas import model, api, rest
-    server.init(stop_consoles=True)
+    server.init()
+    server.check_db_schema()
```

Comment:
Is there a strong reason to run check_db_schema() here instead of within server.init()?